src/storybook/stories/plugins/PluginsListPage.tsx

```diff
@@ -0,0 +1,27 @@
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+import { pageListProps } from "../../../fixtures";
+import PluginsListPage, {
+  PluginsListPageProps
+} from "../../../plugins/components/PluginsListPage";
+import { pluginList } from "../../../plugins/fixtures";
+import Decorator from "../../Decorator";
+
+const props: PluginsListPageProps = {
+  ...pageListProps.default,
+  isChecked: () => undefined,
+  onBack: () => undefined,
+  plugins: pluginList,
+  selected: undefined,
```

Comment:
We should drop bulk actions here, but just for you to know: there is a fixture `listActionsProps` that would help a bit here